src/Routes/README.md

```diff
@@ -1,3 +1,206 @@
 ## Routes
 
-Examples can be found in: `wp-content/plugins/core/src/Routes`
+Examples can be found in: `wp-content/plugins/core/src/Routes/`
+
+### Registering a route
+#### REST Route
+Create a new class in `wp-content/plugins/core/src/Routes/` that extends `Tribe\Libs\Routes\Abstract_REST_Route`
+
+    <?php declare( strict_types=1 );
```

Comment:
This should be `<?php declare(strict_types=1);` one day PHPCS will complain about it, same for the below examples.